### ISSUE TITLE
Possible fix for media_min_like

### ISSUE THREAD

**Issue by DesignBySite** [instabot-py/instabot.py#2479]
Check this or we will delete your issue. (fill in the checkbox with an X like so: [x])
- [x] I have searched for other issues with the same problem or similar feature requests. 

#### Select one:
- [x] Bug
- [ ] Feature Request
- [ ] Technical Help (Please don't create an issue, join the facebook group or telegram channel)
- [ ] Other

#### Environment
- [ ] Are you hosting the bot on a VPS or other Cloud hosting?
- [ ] Are you running the code on your phone 
- [x] If so, have you tried running the code on your local machine?

```
Include more details about the environment
```

#### Operating System? (include version)
- [ ] macOS
- [x] Windows
- [ ] Rasbian Linux Raspberry PI (include flavour)
- [ ] Linux (include flavour)
- [ ] Android (include terminal)

```
include the output of 'uname -a'
Output: MINGW64_NT-10.0-17763 PC12635W 3.0.7-338.x86_64 2019-07-03 15:16 UTC x86_64 Msys
```

#### Python Version Requirement
- [x] I am using Python 3.6 or above

#### Exact Python Version?
```
Include here the response of 'python --version' or 'python3 --version'
Python 3.8.0
```

#### Pip Version?
```
Include here the response of 'pip --version' or 'pip3 --version'
pip 19.2.3 from c:\python3\lib\site-packages\pip (python 3.8)
```

#### Instabot Version?
```
Include here the response of 'instabot-py --version'
Installed version 0.5.16
```

#### How you use the instabot
- [x] instabot-py -c my-configfile.yml (my-configfile.yml is an example you may have a different file )
- [ ] instabot-py --login LOGIN --password PASSWORD
- [ ] instabot-py 
- [ ] example.py (Deprecated, please install newer version) 

#### Description of your issue

```
InstaBot - DEBUG - Will not like this media: number of likes 0 does not meet media_min_like requirements.
- In my config file i have media_min_like: 0
- it seems in the if not statement in instabot.py, it is failing.

 def verify_media(self, media):
        # verify media_min_like requirements
        like_count = media['node']['edge_liked_by']['count']
        if not (self.media_min_like and like_count >= self.media_min_like):
            self.logger.debug(f"Will not like this media: number of likes "
                              f"{like_count} does not meet media_min_like "
                              f"requirements min like is set to {self.media_min_like}") <--- I added to verify media_min_like
            return False

- Output was: InstaBot - DEBUG - Will not like this media: number of likes 0 does not meet media_min_like requirements min like is set to 0

- FIX:
   def verify_media(self, media):
        # verify media_min_like requirements
        like_count = media['node']['edge_liked_by']['count']
        if not like_count >= self.media_min_like: <--- Fix is here
            self.logger.debug(f"Will not like this media: number of likes "
                              f"{like_count} does not meet media_min_like "
                              f"requirements")
            return False
- Also do the same for media_max_like as it has same issue
```

#### Config

```
accept_language: en-US,en;q=0.5
ban_sleep_time: 10800
comment_list:
- - this
  - the
  - your
- - photo
  - picture
  - pic
  - shot
  - snapshot
- - is
  - looks
  - feels
  - is really
- - great
  - super
  - good
  - very good
  - good
  - wow
  - WOW
  - cool
  - GREAT
  - magnificent
  - magical
  - very cool
  - stylish
  - beautiful
  - so beautiful
  - so stylish
  - so professional
  - lovely
  - so lovely
  - very lovely
  - glorious
  - so glorious
  - very glorious
  - adorable
  - excellent
  - amazing
- - .
  - ..
  - '...'
  - '!'
  - '!!'
  - '!!!'
comments_per_day: 0
follow_attempts: 10
follow_per_day: 0
follow_per_run: 0
follow_time: 259200
follow_time_enabled: true
like_followers_per_run: 0
like_per_day: 296
like_per_run: 296
login: ********
max_like_for_one_tag: 15
media_max_like: 0
media_min_like: 0
password: ********
proxies: {}
start_at_h: 0
start_at_m: 0
tag_list:
- famous
- sunday
- dog
time_till_unlike: 259200
unfollow_break_max: 30
unfollow_break_min: 15
unfollow_everyone: false
unfollow_inactive: true
unfollow_not_following: true
unfollow_per_day: 0
unfollow_probably_fake: true
unfollow_recent_feed: true
unfollow_selebgram: false
unfollow_whitelist: []
unlike_per_day: 0
unwanted_username_list: []
user_blacklist: {}
user_max_follow: 0
user_min_follow: 0
window_check_every: 300


```

**Comment by Nadastique** [comment]
same here

**Comment by nickgryg** [comment]
The bug has been fixed in version 0.5.17.
